Directory.Build.targets

```diff
@@ -1,5 +1,33 @@
 <Project>
 
+  <PropertyGroup>
+    <!--
+    For non-SDK projects that import this file and then import Microsoft.Common.targets,
+    tell Microsoft.Common.targets not to import Directory.Build.targets again
+    -->
+    <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
+  </PropertyGroup>
+
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(SkipImportArcadeSdkFromRoot)' != 'true'" />
 
+  <!-- Adds Nullable annotation attributes to netstandard <= 2.0 builds -->
```

Comment:
Is this relevant to anything outside libraries?

Reply:
Any library outside libraries that wants to enable nullable annotations which doesn't target .NET Core >= 3.0 benefits from this. I think there are cases in the other partitions where this might be useful.

Reply:
What are those cases? I expect that we will move all libraries that ship as netstandard targeting NuGet packages to live under libraries.

Reply:
That's a good argument. Will move it back. I was thinking about enabling this scenario just entirely in the repo but if there's no use case for it then this might be over-engineering.